From 2ca933f4cd51b7cd5e6c8512567852de8f09d088 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Tue, 18 Nov 2025 15:24:31 +0100
Subject: [PATCH 030/211] HACK series: for working qdsp6
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Unfortunately, there are multiple slightly different versions used across various forks, we’ve never really converged on a single one.

Mainline: doesn’t make sense in the current state. The current design with the q6voiced service required in userspace has too many integration issues, it conflicts with all other ongoing efforts to improve call audio support in standard userspace components like PipeWire. This is partly also why call audio tends to be unreliable.

# Describe the purpose of this series. The information you put here
# will be used by the project maintainer to make a decision whether
# your patches should be reviewed, and in what priority order. Please be
# very detailed and link to any relevant discussions or sites that the
# maintainer can review to better understand your proposed changes. If you
# only have a single patch in your series, the contents of the cover
# letter will be appended to the "under-the-cut" portion of the patch.

# Lines starting with # will be removed from the cover letter. You can
# use them to add notes or reminders to yourself. If you want to use
# markdown headers in your cover letter, start the line with ">#".

# You can add trailers to the cover letter. Any email addresses found in
# these trailers will be added to the addresses specified/generated
# during the b4 send stage. You can also run "b4 prep --auto-to-cc" to
# auto-populate the To: and Cc: trailers based on the code being
# modified.

Signed-off-by: David Heidelberg <david@ixit.cz>

--- b4-submit-tracking ---
# This section is used internally by b4 prep for tracking purposes.
{
  "series": {
    "revision": 1,
    "change-id": "20251118-qdsp6-2a3e5a72a045",
    "prefixes": []
  }
}
-- 
2.55.0

